### PR TITLE
Fix in order to work with new Wikipedia format

### PR DESCRIPTION
**Pull request by came88**
Now the extension should work in any math rendering mode.
In MathML mode this fix does not use the MathML code but the original TeX inside an &lt;annotation&gt; inside &lt;math&gt;.

### CODE REVIEW

File: content.js [hunk 1]
```diff
@@ -6,23 +6,32 @@ function compareRight(s1, s2) {
...
 
 function replaceUnbalancedBraces(text) {
+  console.log('Balancing ' +  text);
   var stack = [];
   var ret = text.split('');
 
   for (var i = 0; i < text.length; ++i) {
-    if (text[i] == '(') {
+    if (text[i] == '\\') {
+      // handle special cases such as \left( or \right)
```
Comment (bgromov) [inline content.js]: I was wondering where is the problem for quite a long time, thanks!

File: content.js [hunk 2]
```diff
@@ -6,7 +6,6 @@ function compareRight(s1, s2) {
 var isWikiwand = compareRight(document.location.hostname, 'wikiwand.com');
 
 function replaceUnbalancedBraces(text) {
-  console.log('Balancing ' +  text);
```
Comment (bgromov) [inline content.js]: Won't it make sense to remove other log messages as well? E.g. "Replacing PNG images with MathJax..."

File: content.js (hunk 1 above)
Comment (came88) [inline content.js]: Unfortunately it doesn't fix all formulas, like the one in https://www.wikiwand.com/it/Pi_greco
`{\displaystyle ={\frac {1}{r}}{\int _{-r}^{r}{\sqrt ((\frac {x^{2)){r^{2}-x^{2}}}+1}}\,dx}}` should become
`{\displaystyle ={\frac {1}{r}}{\int _{-r}^{r}{\sqrt {{\frac {x^{2}}{r^{2}-x^{2}}}+1}}\,dx}}` so a couple of `(` should be `{`.
I can't understand why there are these TeX errors in wikiwand

File: content.js (hunk 2 above)
Comment (came88) [inline content.js]: Ok, I'm going to remove them all

File: content.js (hunk 1 above)
Comment (bgromov) [inline content.js]: Yes, these are the nasty ones. I have reported this issue to Wikiwand support quite a long time ago, but received no feedback, so I tried to fix it manually.

**Pull request merged**